### DISCUSSION TITLE
Option to lobotomize other entities

### DISCUSSION THREAD

**Discussion by 0x000006** [PurpurMC/Purpur#1551] [Feature Requests]
For example lobotomize sheep when they are in a wool farm or iron golems when they are in a slime farm cage.

**Comment by Onako2** [comment]
How do you mean? You mean that the entities should be frozen?

**Reply by 0x000006** [reply]
They should have some of their AI like pathfinding and looking at entities disabled because that causes lag but some parts enabled like sheep eating grass.
